doc/README.open_ce_build.md

```diff
@@ -172,6 +173,8 @@ optional arguments:
   --conda_pkg_format CONDA_PKG_FORMAT
                         Conda package format to be used, such as "tarball" or
                         "conda". (default: conda)
+  --ppc_arch PPC_ARCH   Power Architecture to build for, such as "p9" or "p10". Use p10 for power10 enabled builds.
```

Comment:
> such as "p9" or "p10"

I suggest to write the actual supported values. is it p9, p10, or also p8 ?

Reply:
Actually in code, it's either p9 or p10. "p9" means "p8" as well as "p9" type of builds (our type1 builds).

Reply:
Ok, then it should say. 
Power Architecture to build for. Values: `p9` or `p10`.  `p9`: Libraries can be used on Power8, Power9 and Power 10, but do not use MMA acceleration. `p10`: Libraries can be used on Power9 and Power10, and use MMA acceleration on Power10.